### PR TITLE
Feature/beom init

### PR DESCRIPTION
**Pull request by bbeomjin**
## What does this PR do?
fix errors under the python=3.8 and tensorflow=2.5

## How to test this? 
-  run "run.py"

## Does anything else need to be done?
- nothing

### CODE REVIEW

File: module/common/replaybuffer.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def __init__(
         """
         self.buffer = deque(maxlen=capacity)
         self.buffer_count = 0
+        self.buffer_size = capacity
```
Comment (RektPunk) [inline module/common/replaybuffer.py:30]: why you add this?

File: module/schemas/common.py [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+import numpy as np
 
 
 class Transition(BaseModel):
-    state: float
-    action: float
-    reward: float
-    next_state: float
-    done: float
+    state: np.ndarray
```
Comment (RektPunk) [inline module/schemas/common.py:7]: isn't it list of np.ndarray?
For example,
```
state: List[np.ndarray]
```

File: module/schemas/common.py [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
+    action: np.ndarray
+    reward: int
+    next_state: np.ndarray
+    done: bool
+
+    class Config:
```
Comment (RektPunk) [inline module/schemas/common.py:13]: How bout set this in Optional?

**Review by RektPunk — CHANGES_REQUESTED** [review]

File: module/schemas/common.py (hunk 2 above)
Comment (RektPunk) [reply module/schemas/common.py:7]: Also, float and np.ndarray are compatible.

**Review by RektPunk — CHANGES_REQUESTED** [review]

File: module/common/replaybuffer.py (hunk 1 above)
Comment (bbeomjin) [reply module/common/replaybuffer.py:30]: Because buffer_size variable is used in "add()" method in line 38

**Review by RektPunk — APPROVED** [review]
gogo

**Comment by rockgoat95** [comment]
엄지척 bb

**Pull request merged**